.env_example

```diff
@@ -26,8 +26,7 @@ META_TXN_RELAY_PRIVATE_KEYS=
 META_TXN_RELAY_EXPECTED_MINED_SEC=
 ENABLE_PROMETHEUS_METRICS=
 PROMETHEUS_PORT=
-RFQT_API_KEY_WHITELIST=
-PLP_API_KEY_WHITELIST=*
+RFQT_API_KEY_WHITELIST_JSON=
```

Comment:
Shouldn't there be `PLP_API_KEY_WHITELIST_JSON` as well?

Reply:
Good call, will add!